### ISSUE TITLE
Create a blank issue form with no dependency

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/website#2954]
### Overview
As a developer, I want to create a blank issue form for issues with no dependencies as the forms (compared to templates) in some cases, are easier and more intuitive in creating a new issue.

### Action Items
- [x] Review [blank issue form](https://github.com/hackforla/website/blame/gh-pages/.github/ISSUE_TEMPLATE/blank-issue-form.yml) yml file as well as the [blank issue form](https://github.com/hackforla/website/issues/new?assignees=&labels=&template=blank-issue-form.yml)
- [x] In a new topic branch of your own repo, create a new file in the directory `website/.github/ISSUE_TEMPLATE` titled "blank-issue-form-with-no-dependency.yml"
- [x] Copy and paste all the code from [blank issue form](https://github.com/hackforla/website/blame/gh-pages/.github/ISSUE_TEMPLATE/blank-issue-form.yml) yml file to your newly created yml file except lines 26-38.
- [x] Change the title and description in lines 1 and 2 (in your newly created yml file):

from
```
name: 'Blank Issue '
description: 'Standard HackforLA issue template '
```
to
```
name: 'Blank Issue Form with No Dependency'
description: 'Standard HackforLA issue form with no dependency'
```
- [x] Please add a preview screenshot of the newly created form.
- [x] Push the changes and make a pull request
- [x] Once the pull request associated with this issue is approved and merged, please update and edit issue #2904 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please do the following for issue #2904:
    - [ ] Move it to the New Issue Approval column
    - [ ] Remove the "Dependency" label from it
    - [ ] Add a "ready for dev lead" label to it

### Resources/Instructions
- [Blanks issue form yml file](https://github.com/hackforla/website/blame/gh-pages/.github/ISSUE_TEMPLATE/blank-issue-form.yml)
- [Blank issue form](https://github.com/hackforla/website/issues/new?assignees=&labels=&template=blank-issue-form.yml)
- Directory where file needs to be created `website/.github/ISSUE_TEMPLATE`

**Comment by carolemlago** [comment]
Availability: 2-5pm Wed/Thu
ETA: 6pm Thursday

**Comment by carolemlago** [comment]
Availability: 10AM - 1PM FRI (waiting on initial approval and merge to complete dependency issues)
ETA: 6pm Friday
